Mathlib/AlgebraicTopology/TopologicalSimplex.lean

```diff
@@ -49,6 +50,7 @@ def toTopMap {x y : SimplexCategory} (f : x ⟶ y) (g : x.toTopObj) : y.toTopObj
       simp [Finset.eq_univ_iff_forall]
     · apply Set.pairwiseDisjoint_filter⟩
 
+open Classical in
```

Comment:
Same question here.

Reply:
It's for rewriting from left to right, as above. In general, if decidability only appears on the right of an equality, I went for `open Classical in` for this reason.